DataFormats/SiStripCluster/interface/SiStripApproximateCluster.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DATAFORMATS_SISTRIPAPPROXIMATECLUSTER_H
+#define DATAFORMATS_SISTRIPAPPROXIMATECLUSTER_H
+
+#include <numeric>
+#include <cmath>
+#include <iostream>
+#include <iomanip>
+
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+
+class SiStripApproximateCluster {
+public:
+  SiStripApproximateCluster() {}
+
+  explicit SiStripApproximateCluster(float barycenter, uint8_t width, float avgCharge) {
+    barycenter_ = barycenter;
+    width_ = width;
+    //if(width_>0x3F) width_=0x3F;
```

Comment:
I think these commented out lines here ande below can be removed?

Reply:
I was keeping them for future reference but sure, removed

Reply:
done